clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -919,6 +922,8 @@ class SyclKernelDeclCreator
         FuncType, Ctx.getTrivialTypeSourceInfo(Ctx.VoidTy), SC_None);
     FD->setImplicitlyInline(IsInline);
     setKernelImplicitAttrs(Ctx, FD, Name);
+    if (IsSIMDKernel)
```

Comment:
This should probably be in the setKernelImplicitAttrs above.

Reply:
make sense